RevenueCatUI/Plugins/Android/RevenueCatUI.androidlib/src/main/java/com/revenuecat/purchasesunity/ui/RevenueCatUI.java

```diff
@@ -32,4 +41,17 @@ public static void sendPaywallResult(String result) {
             Log.e(TAG, "Error sending paywall result: " + e.getMessage());
         }
     }
-}
\ No newline at end of file
+
+    public static void sendCustomerCenterResult(String result) {
```

Comment:
Is this ever called at all? Do we need it?

Reply:
Nope, this is a leftover of the result api